CHANGELOG.md

```diff
@@ -55,6 +55,7 @@ Mark  items as `Added`, `Changed`, `Fixed`, `Removed`, `Untested Features`, `Upc
 * Added environment variable `CUSTOM_SUBMISSION_URL`, `CUSTOM_SUBMISSION_ENABLED` for support form adapter.
 * Added environment variables `DRAFT_ENABLED`, `DRAFT_POLLING_RATE` to manage draft feature. 
 * Added environment variable `EXPORT_PDF_ENABLED`for pdf service.
+* Added environment variable `PUBLIC_WORKFLOW_ENABLED` for enabling public workflow creation
```

Comment:
can mention this is for multitenant specific

Reply:
done

Reply:
done